burst_detector/utils.py

```diff
@@ -19,13 +21,66 @@
 logger = logging.getLogger("burst-detector")
 
 
+def parse_args() -> dict[str, Any]:
```

Comment:
can you rename this so that it's clear from the function name that this is the args for plot_units

Reply:
fixed